[2020-02-25 13:46:39,775] {taskinstance.py:655} INFO - Dependencies all met for <TaskInstance: pouli.step__main_step__test__1 2020-02-25T13:46:28.342158+00:00 [queued]>
[2020-02-25 13:46:39,819] {taskinstance.py:655} INFO - Dependencies all met for <TaskInstance: pouli.step__main_step__test__1 2020-02-25T13:46:28.342158+00:00 [queued]>
[2020-02-25 13:46:39,819] {taskinstance.py:866} INFO - 
--------------------------------------------------------------------------------
[2020-02-25 13:46:39,819] {taskinstance.py:867} INFO - Starting attempt 1 of 1
[2020-02-25 13:46:39,819] {taskinstance.py:868} INFO - 
--------------------------------------------------------------------------------
[2020-02-25 13:46:39,878] {taskinstance.py:887} INFO - Executing <Task(BashOperator): step__main_step__test__1> on 2020-02-25T13:46:28.342158+00:00
[2020-02-25 13:46:39,896] {standard_task_runner.py:53} INFO - Started process 5477 to run task
[2020-02-25 13:46:40,356] {logging_mixin.py:112} INFO - Running %s on host %s <TaskInstance: pouli.step__main_step__test__1 2020-02-25T13:46:28.342158+00:00 [running]> df49cf1bf754
[2020-02-25 13:46:40,480] {bash_operator.py:82} INFO - Tmp dir root location: 
 /tmp
[2020-02-25 13:46:40,483] {bash_operator.py:105} INFO - Temporary script location: /tmp/airflowtmpnt5h65wz/step__main_step__test__1w7j2cqiv
[2020-02-25 13:46:40,484] {bash_operator.py:115} INFO - Running command: 
OBC_START=$(eval "declare")
:
# Insert the BASH commands for this step.
# You can use the variable ${OBC_WORK_PATH} as your working directory.

touch /usr/local/airflow/WORK/aaa.bbb

echo "Hello"
OBC_CURRENT=$(eval "declare")
comm -3 <(echo "$OBC_START" | grep -v "_=") <(echo "$OBC_CURRENT" | grep -v OBC_START | grep -v PIPESTATUS | grep -v "_=") > ${OBC_WORK_PATH}/step__main_step__test__1__1.sh

[2020-02-25 13:46:40,501] {bash_operator.py:122} INFO - Output:
[2020-02-25 13:46:40,518] {bash_operator.py:126} INFO - touch: cannot touch '/usr/local/airflow/WORK/aaa.bbb': No such file or directory
[2020-02-25 13:46:40,519] {bash_operator.py:126} INFO - Hello
[2020-02-25 13:46:40,531] {bash_operator.py:126} INFO - /tmp/airflowtmpnt5h65wz/step__main_step__test__1w7j2cqiv: line 11: /usr/local/airflow/WORK/step__main_step__test__1__1.sh: No such file or directory
[2020-02-25 13:46:40,545] {bash_operator.py:130} INFO - Command exited with return code 1
[2020-02-25 13:46:40,586] {taskinstance.py:1128} ERROR - Bash command failed
Traceback (most recent call last):
  File "/usr/local/lib/python3.7/site-packages/airflow/models/taskinstance.py", line 966, in _run_raw_task
    result = task_copy.execute(context=context)
  File "/usr/local/lib/python3.7/site-packages/airflow/operators/bash_operator.py", line 134, in execute
    raise AirflowException("Bash command failed")
airflow.exceptions.AirflowException: Bash command failed
[2020-02-25 13:46:40,590] {taskinstance.py:1185} INFO - Marking task as FAILED.dag_id=pouli, task_id=step__main_step__test__1, execution_date=20200225T134628, start_date=20200225T134639, end_date=20200225T134640
[2020-02-25 13:46:49,707] {logging_mixin.py:112} INFO - [2020-02-25 13:46:49,706] {local_task_job.py:103} INFO - Task exited with return code 1
